src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,46 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = (
+  { tabs,
+    visibleTab,
+    selectedTabId,
+    onTabSelected },
+) => {
+  let selectTheTab = selectedTabId;
+
+  if (!tabs.some(tab => tab.id === selectTheTab)) {
+    selectTheTab = tabs[0].id;
+  }
```

Comment:
Not sure if you need this

Reply:
Yes, I need this. There is one test that says that the page should highlight the first tab if the input for selected tab is invalid. In particular tests it is something like 'addschs'... WIthout this part I`m not sure how to pass this test